src/queues/job-queue.js

```diff
@@ -85,7 +84,7 @@ const initQueue = () => {
   });
 
   console.info('deleteActivityProcessQueue job is working!');
-  deleteActivityProcessQueue.process(deleteActivityProcessJob);
+  deleteActivityProcessQueue.process(removeActivityProcessJob);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The queue name `deleteActivityProcessQueue` hasn't been updated to reflect the new job name `removeActivityProcessJob`. This could lead to a runtime error if `deleteActivityProcessQueue` doesn't exist or isn't defined elsewhere in the code.
```diff
- deleteActivityProcessQueue.process(removeActivityProcessJob);
+ removeActivityQueue.process(removeActivityProcessJob);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->